sdk/storage/azure-storage-blob/src/main/java/com/azure/storage/blob/implementation/models/QueryFormat.java

```diff
@@ -5,49 +5,53 @@
 package com.azure.storage.blob.implementation.models;
 
 import com.azure.core.annotation.Fluent;
-import com.fasterxml.jackson.annotation.JsonProperty;
-import com.fasterxml.jackson.dataformat.xml.annotation.JacksonXmlRootElement;
-
-/** The QueryFormat model. */
-@JacksonXmlRootElement(localName = "QueryFormat")
+import com.azure.core.util.CoreUtils;
+import com.azure.xml.XmlReader;
+import com.azure.xml.XmlSerializable;
+import com.azure.xml.XmlToken;
+import com.azure.xml.XmlWriter;
+import javax.xml.namespace.QName;
+import javax.xml.stream.XMLStreamException;
+
+/**
+ * The QueryFormat model.
+ */
 @Fluent
-public final class QueryFormat {
+public final class QueryFormat implements XmlSerializable<QueryFormat> {
     /*
      * The quick query format type.
      */
-    @JsonProperty(value = "Type", required = true)
     private QueryFormatType type;
 
     /*
      * Groups the settings used for interpreting the blob data if the blob is delimited text formatted.
      */
-    @JsonProperty(value = "DelimitedTextConfiguration")
     private DelimitedTextConfiguration delimitedTextConfiguration;
 
     /*
      * json text configuration
      */
-    @JsonProperty(value = "JsonTextConfiguration")
     private JsonTextConfiguration jsonTextConfiguration;
 
     /*
      * Groups the settings used for formatting the response if the response should be Arrow formatted.
      */
-    @JsonProperty(value = "ArrowConfiguration")
     private ArrowConfiguration arrowConfiguration;
 
     /*
      * parquet configuration
      */
-    @JsonProperty(value = "ParquetTextConfiguration")
     private Object parquetTextConfiguration;
```

Comment:
Should `parquetTextConfiguration` have a "dummy" type added into the Swagger so we don't need to hack the object creation in `fromXml` and serialization in `toXml`? This would also allow the type to grow in the future if actual configurations are added to Parquet.